server/src/internalClusterTest/java/org/elasticsearch/action/search/TransportSearchIT.java

```diff
@@ -159,6 +161,7 @@ public void testLocalClusterAlias() {
     }
 
     public void testAbsoluteStartMillis() {
+        TaskId takeId = new TaskId("node", randomNonNegativeLong());
```

Comment:
`parentTaskId` ?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/66206/commits/646aeab9ed596e00360112b72e628839aa98a47c